hawkmoth/parser.py

```diff
@@ -34,7 +34,7 @@
 from dataclasses import dataclass
 
 from clang.cindex import TokenKind, CursorKind, TypeKind
-from clang.cindex import StorageClass, AccessSpecifier
+from clang.cindex import StorageClass, AccessSpecifier, ExceptionSpecificationKind
```

Comment:
Side note, what do you think of this style of imports:

```
from clang.cindex import (AccessSpecifier,
                          CursorKind,
                          Diagnostic,
                          Index,
                          StorageClass,
                          TokenKind,
                          TranslationUnit,
                          TypeKind)

I think our cindex imports are getting a bit unweildy. Usually I'd prefer just `from clang import cindex` and then then using the `cindex.TokenKind` syntax everywhere, but here that gets too verbose in code, I think.

And absolutely off-topic for the change at hand!

```

Reply:
Yeah, I kind of like it. I agree with you too, not ideal, but too verbose in our case otherwise.

Thanks for the merge!